daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Transaction.scala

```diff
@@ -792,14 +792,9 @@ object Transaction {
   type ActionNode = Node.GenActionNode
   type LeafNode = Node.LeafOnlyActionNode
 
-  /** (Complete) transactions, which are the result of interpreting a
-    * ledger-update. These transactions are consumed by either the
-    * scenario-interpreter or the Daml-engine code. Both of these
-    * code-paths share the computations for segregating the
-    * transaction into party-specific ledgers and for computing
-    * divulgence of contracts.
-    */
+  @deprecated("use com.daml.transaction.VersionedTransaction", since = "1.8.0")
```

Comment:
do you mean 1.18? Retroactively deprecating since 1.8 seems a bit weird.

Reply:
Good catch. There is the same in issue in `remy-lf-nodes` branch. 
I will fix that.